src/custom-types/cleanbc/cleanbc-actions.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
could we have used the acf plugin?

Reply:
Possibly. I was attempting a few things with this: to limit dependencies on third party plugins, to provide the ability to suppress site specific features (CPT, taxonomy, block filters, etc.) and, to enable all from a base installation with a clean DB.